sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -59,13 +64,14 @@ case class IdentifiedWork(
   lettering: Option[String] = None,
   createdDate: Option[Period] = None,
   subjects: List[Subject[Displayable[AbstractConcept]]] = Nil,
-  contributors: List[Contributor[Displayable[AbstractAgent]]] = Nil,
   genres: List[Genre[Displayable[AbstractConcept]]] = Nil,
+  contributors: List[Contributor[Displayable[AbstractAgent]]] = Nil,
   thumbnail: Option[Location] = None,
   items: List[IdentifiedItem] = Nil,
   production: List[ProductionEvent[Displayable[AbstractAgent]]] = Nil,
   language: Option[Language] = None,
   dimensions: Option[String] = None,
+  version: Int,
```

Comment:
Minor, but I tend to prefer having all members with no default value at the beginning of the list so that it's easier to spot what's the minimum amount of stuff that must be provided

Reply:
It’s getting quite